tests/components/deconz/test_binary_sensor.py

```diff
@@ -701,10 +678,15 @@ async def test_add_new_binary_sensor_ignored_load_entities_on_service_call(
     assert hass.states.get("binary_sensor.presence_sensor")
 
 
+@pytest.mark.parametrize(
+    "config_entry_options", [{CONF_MASTER_GATEWAY: True, CONF_ALLOW_NEW_DEVICES: False}]
+)
 async def test_add_new_binary_sensor_ignored_load_entities_on_options_change(
     hass: HomeAssistant,
     entity_registry: er.EntityRegistry,
-    aioclient_mock: AiohttpClientMocker,
+    config_entry_setup: ConfigEntry,
+    deconz_payload: dict[str, Any],
+    mock_requests: Callable[[str], None],
     mock_deconz_websocket,
```

Comment:
https://github.com/home-assistant/core/pull/120967#discussion_r1665452557

Reply:
Will do in a separate PR